docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,97 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`MicroK8s is installed <set_microk8s>`.
+* :ref:`DSS CLI is installed <install_DSS_CLI>`.
+* The Intel GPU is recognised by your machine. This can be confirmed by running `intel_gpu_top` from the `intel-gpu-tools` package. For discrete GPUs on older versions of Ubuntu like 22.04, you may need to perform additional steps like install a HWE kernel.
+
+Install the kubectl snap
+---------------------------
+First, install the `kubectl` snap package:
+
+.. code-block:: bash
+				
+  sudo snap install kubectl -–classic
+
+Create a configuration file so that `kubectl` can communicate with MicroK8s by running:
+
+.. code-block:: bash
+				
+  mkdir -p ~/.kube
+  microk8s config > ~/.kube/config
+
+Enable the Intel GPU plugin in your Kubernetes cluster 
+------------------------------------------------------
+
+To ensure DSS can utilise Intel GPUs, you have to enable the Intel GPU plugin in your Kubernetes cluster. The plugin uses the `Intel device plugins for Kubernetes`_ to detect whether an Intel GPU exists on a machine. You can see the `Installation with NFD`_ page for more details.
+
+To enable the plugin, you must download the appropriate `kustomization.yaml` files from Intel’s device plugins for Kubernetes repository, and then apply them on your MicroK8s cluster.
+
+First, build the YAML files for Node Feature Discovery, Node Feature Rules, and the GPU plugin, by running:
+
+.. code-block:: bash
+
+  VERSION=v0.30.0
+  kubectl kustomize https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd?ref=${VERSION} > node_feature_discovery.yaml
+  kubectl kustomize https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd/overlays/node-feature-rules?ref=${VERSION} > node_feature_rules.yaml
+  kubectl kustomize https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/gpu_plugin/overlays/nfd_labeled_nodes?ref=${VERSION} > gpu_plugin.yaml
+
+To allow multiple containers to utilise the same GPU, run:
+
+.. code-block:: bash
+				
+  sed -i 's/enable-monitoring/enable-monitoring\n        - -shared-dev-num=10/' gpu_plugin.yaml
+
+Then, apply the created YAML files to your MicroK8s cluster by running:
```

Comment:
```suggestion
Then, apply the downloaded YAML files to your MicroK8s cluster by running:
```
Weren't they downloaded?

Reply:
Well
We download `kustomization.yaml` files, and we apply `kubectl kustomize` on them, and then we redirect the output to .yaml files. So I think in this case "create" is slightly more fitting.